src/main/java/com/google/edith/servlets/ReceiptFileHandlerServlet.java

```diff
@@ -32,6 +34,10 @@
 @WebServlet("/receipt-file-handler")
 public final class ReceiptFileHandlerServlet extends HttpServlet {
 
+  private BlobKey fileBlobKey;
+  private String expenditureName;
```

Comment:
Why declare these here rather than initialize in the method where they are used directly?

Reply:
same as above, this file is an unmerged dependency